addons/script-ide/quickopen/quick_open_panel.gd

```diff
@@ -83,10 +83,6 @@ func schedule_rebuild():
 	is_rebuild_cache = true
 
 func on_show():
-	if (search_option_btn.selected != 0):
```

Comment:
This is still needed for the `CheckBox`, so that the `pressed` state is reset when needed